src/krux/pages/login.py

```diff
@@ -247,8 +247,9 @@ def load_key_from_qr_code(self):
 
             if not words:
                 try:
-                    data_bytes = data.encode() if isinstance(data, str) else data
-
+                    data_bytes = (
+                        data.encode("latin-1") if isinstance(data, str) else data
```

Comment:
Did you confirm that this (`data.encode("latin-1")`) works on the device? I noticed some differences between Micropython and Python 3 when it comes to encodings around strings and bytes... some things work in the simulator but not on the devices, and vice versa.

Reply:
yes, I came to this after looking for something that would work in both